apps/admin-portal/src/components/claims/dynamic-fields.tsx

```diff
@@ -132,7 +136,7 @@ export const DynamicField = (props: DynamicFieldPropsInterface): React.ReactElem
                                                         <Field
                                                             type={ keyType }
                                                             placeholder=""
-                                                            required={ true }
+                                                            required={ (requiredField === undefined) ? true : requiredField }
```

Comment:
Add default props to the component so we can avoid the `undefined` check.

```js
DynamicField.defaultProps = {
    requiredField: true
}
```

Reply:
Don't you think it is better if `requiredField` is `false` by default. Prolly, @inthirakumaaran tried to ensure that adding `requiredField` to the `DynamicFields` component didn't break the Claims section. Better if `requiredField` can be passed as a prop from the Claims components to resolve this issue. 

The following files will have to be modified in the `src/components/claims` directory:
-`edit/edit-additional-properties-local-claims.tsx`
-`edit/edit-mapped-attributes-local-claims.tsx`
-`wizard/mapped-attributes.tsx`

Just pass `requiredField={ true }` to the `DynamicFields` component.